test/workbox-precaching/static/addToCacheList.html

```diff
@@ -0,0 +1,24 @@
+<html>
```

Comment:
I think I've see a few other of these `static/*.html` files checked in now. How are they used?

Reply:
At the moment they are just manual tests.

If you run gulp build and start a static server, you can see what the end developer would see (It's where the screenshots in this and the log PR came from). Happy to remove. I am mindful that they could break without anyone knowing.